app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="shopping-list-wrapper">
+  <div class="shopping-list-container">
+    <h3 class="shopping-list-title">Shopping List</h3>
+    <div class="shopping-list-summary">
+      <div class="shopping-list-summary-item">
+        <% totalQuantity = 0 %>
+        <% @recipes.each do |recipe_food| %>
+          <% totalQuantity += recipe_food.quantity %>
+        <% end %>
+        <p class="shopping-list-info">Amount of food items to buy: <strong><%= totalQuantity %></strong></p>
+      </div>
+      <div class="shopping-list-summary-item">
+        <% totalValue = 0 %>
+        <% @recipes.each do |recipe_food| %>
+          <% totalValue += recipe_food.food.price * recipe_food.quantity %>
+        <% end %>
+        <p class="shopping-list-info">Total value of food needed: $<strong><%= totalValue %></strong></p>
+      </div>
+    </div>
+    <table class="shopping-list-table">
+      <thead>
+        <tr>
+          <th>Food</th>
+          <th>Quantity</th>
+          <th>Unit Price</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @recipes.each do |recipe_food| %>
+          <tr>
+            <td class="shopping-list-table-cell"><%= recipe_food.food.name %></td>
+            <td class="shopping-list-table-cell"><%= pluralize(recipe_food.quantity, recipe_food.food.measurement_unit) %></td>
+            <td class="shopping-list-table-cell">$<%= recipe_food.food.price %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
```

Comment:
-- [ Optional ] Good job implementing the changes from the previous reviewer, I would like to mention a few things that could improve your design. Currently, your design varies with the project wireframe in the following ways.
   
   - [ ] The shopping list is not styled the same way it has been styled in the project wireframe. If you look at the project wireframe it is `zebra` stripped, please see the screenshot below for more details on this. I recommend you fix this in order to improve your project.

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/general_shopping_list.png?token=GHSAT0AAAAAABTYBUDYPJYL4L6ODBIW2HESZFAFHMA)

## Your design
![Screenshot from 2023-07-01 15-28-55](https://github.com/ngeelani48/Recipe-app/assets/16392046/a0ccb4a0-051b-4828-970e-4a11131e3110)

Reply:
@Felix45, I will change the styles and dimensions of this table too.
Thanks for the approval and suggestions.  💯